source/includes/mongosync-and-oidc.rst

```diff
@@ -0,0 +1,4 @@
+Starting in 1.8.1, you can use ``mongosync`` with OpenID Connect (OIDC)
+to connect to MongoDB clusters running on Microsoft Azure and Google
+Cloud Platform. OIDC is a protocol that allows users to authenticate
+using an external identity provider.
```

Comment:
Another suggestion:
```
Starting in 1.8.1, ``mongosync`` can authenticate via OpenID Connect (OIDC) when connecting to MongoDB clusters that run on Microsoft Azure or Google Cloud Platform.
```
This may touch on internal documentation style, but FWIW I would also, rather than putting a separate sentence in that explains what OIDC is, just link to some document that explains OIDC authentication. (Maybe MongoDB’s own?) I would _think_ that the bulk of users for whom this feature is useful would already know what those terms are. (@pmeredit, thoughs?)

Reply:
We should link to the mongodb workload identity documentation:
https://www.mongodb.com/docs/atlas/workload-oidc/

Or perhaps even the blog:
https://www.mongodb.com/blog/post/mongodb-introduces-workload-identity-federation-database-access

As well as somewhere linking to the uri options for authentication:
https://www.mongodb.com/docs/manual/reference/connection-string-options/#authentication-options